CODEOWNERS

```diff
@@ -1,13 +1,13 @@
 * @Microsoft/onnxruntime
 
 # Python frontend owners
-orttraining/*.py @thiagocrepaldi @spandantiwari @BowenBao @liqunfu @baijumeswani @mrry
```

Comment:
Might drop @mrry as well.

Reply:
Good point!

Reply:
@nkreeger, would you want to be added as a code owner?